docs/usage/information_provider_system.md

```diff
@@ -198,9 +198,9 @@ the [Mouser API page](https://www.mouser.de/api-home/).
 You will receive an API token, which you have to put in the Part-DB env configuration (see below):
 At the registration you choose a country, language, and currency in which you want to get the results.
 
-*Attention*: Currently (January 2024) the mouser API seems to be somewhat broken, in the way that it does not return any
-information about datasheets and part specifications. Therefore Part-DB can not retrieve them, even if they are shown
-at the mouser page. See [issue #503](https://github.com/Part-DB/Part-DB-server/issues/503) for more info.
+*Attention*: Currently (January 2024), the Mouser API seems to be somewhat broken, in the way that it does not return any
+information about datasheets and part specifications. Therefore Part-DB cannot retrieve them, even if they are shown
+on the Mouser page. See [issue #503](https://github.com/Part-DB/Part-DB-server/issues/503) for more info.
```

Comment:
Issue #503 is closed, this information is stale.

Reply:
Although the imperfection of the API still exists...
But I think one should generally not assume that every API produces perfect responses.
Maybe an additional caveat on the limited accuracy of API results should be incorporated.